internal/service/advancedcluster/data_source_advanced_cluster.go

```diff
@@ -329,7 +329,6 @@ func dataSourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.
 		return diag.FromErr(fmt.Errorf(ErrorClusterAdvancedSetting, "replication_specs", clusterName, err))
 	}
 
-	// TODO: CLOUDP-258711 update to use connLatest to call below API
```

Comment:
this comment is outdated, for advanced configuration we have to use old API because of removed properties in the new API.